pkg/status/status_system_probe_unsupported.go

```diff
@@ -0,0 +1,13 @@
+// +build zlib
+
+package status
+
+import (
+	"fmt"
+)
+
+func getSystemProbeStats() map[string]interface{} {
+	return map[string]interface{}{
+		"Errors": fmt.Sprintf("System Probe Unsupported"),
```

Comment:
We should give a better explanation I think. Something like `System Probe is not supported when build without zlib support` ?